Content.Server/Tabletop/TabletopNanopoliaSetup.cs

```diff
@@ -0,0 +1,95 @@
+using JetBrains.Annotations;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Tabletop
+{
+    [UsedImplicitly]
+    public sealed partial class TabletopNanopoliaSetup : TabletopSetup
+    {
+
+        [DataField("FigurineCatPrototype", customTypeSerializer:typeof(PrototypeIdSerializer<EntityPrototype>))]
```

Comment:
dont have CamelCase fields, just have [DataField] it has automatic name
and use EntProtoId for these instead of string

Reply:
If I understand correctly, I did

Reply:
use just [DataField] then
`public EntProtoid FigurineCatPrototype ...`

Reply:
Sorry, I don't understand how to use EntProtoid... I'm too dumb

Reply:
I figured out how to use it!